padle/view/padle_scroll_view.py

```diff
@@ -0,0 +1,105 @@
+from math import ceil
+from typing import Collection, TYPE_CHECKING
+
+from discordmenu.embed.components import EmbedMain, EmbedField
+from discordmenu.embed.base import Box
+from discordmenu.embed.view import EmbedView
+from tsutils.menu.components.footers import embed_footer_with_state
+from tsutils.menu.components.config import UserConfig
+from tsutils.menu.view.view_state_base import ViewStateBase
+from tsutils.query_settings.query_settings import QuerySettings
+from padle.monsterdiff import MonsterDiff
+
+from discordmenu.embed.components import EmbedFooter
+
+from tsutils.tsubaki.links import CLOUDFRONT_URL
+
+TSUBAKI_FLOWER_ICON_URL = CLOUDFRONT_URL + '/tsubaki/tsubakiflower.png'
+
+if TYPE_CHECKING:
+    from dbcog.models.monster_model import MonsterModel
+
+
+class PADleScrollViewState(ViewStateBase):
+    VIEW_STATE_TYPE: str = "PADleScrollView"
+
+    def __init__(self, original_author_id, menu_type, raw_query, monster, cur_day_guesses=[],
+                 current_page=0, extra_state=None, reaction_list=None, current_day: int = 0):
+        super().__init__(original_author_id, menu_type, raw_query,
+                         extra_state=extra_state)
+        self.cur_day_guesses = cur_day_guesses
+        self.cur_day_page_guesses = cur_day_guesses[((current_page) * 5):((current_page + 1) * 5)]
+        self.current_page = current_page
+        self.reaction_list = reaction_list
+        self.current_day = current_day
+        self.monster = monster
+        self.num_pages = ceil(len(cur_day_guesses) / 5)
+
+    def get_cur_fields(self):
+        fields = []
+        for index, item in enumerate(self.cur_day_page_guesses):
+            guessed_mon = MonsterDiff(self.monster, item)
+            fields.append(EmbedField(
+                title=f"**Guess #{5 * (self.current_page) + index + 1}**:",
+                body=Box("\n".join(
+                    [guessed_mon.get_name_line_feedback_text(), guessed_mon.get_other_info_feedback_text(),
+                     guessed_mon.get_awakenings_feedback_text()])),
+            ))
+        return fields
+
+    @classmethod
+    async def do_queries(cls, dbcog, guess_ids):
+        monster_list = []
+        for id in guess_ids:
+            monster_list.append(dbcog.get_monster(int(id)))
+        return monster_list
+
+    def get_pages_footer(self):
+        return "Page " + str(self.current_page + 1) + "/" + str(self.num_pages)
+
+    @classmethod
+    def get_monster_ids_list(cls, monster_list):
+        ids = []
+        for monster in monster_list:
+            ids.append(monster.monster_id)
+        return ids
+
+    def serialize(self):
+        ret = super().serialize()
+        ret.update({
+            'current_page': self.current_page,
+            'reaction_list': self.reaction_list,
+            'current_day': self.current_day,
+            'cur_monster': self.monster.monster_id,
+            'guesses_ids': self.get_monster_ids_list(self.cur_day_guesses),
```

Comment:
you shouldn't need to serialize today's guesses; in the `deserialize` method you have `today_guesses` passed via the data object (and serializing guess_ids limits the number of guesses a user can make)